addons/web_editor/models/ir_qweb.py

```diff
@@ -57,6 +53,14 @@ def render_tag_snippet(self, element, template_attributes, generated_attributes,
         template_attributes['call'] = template_attributes['snippet']
         return self.render_tag_call(element, template_attributes, generated_attributes, qwebcontext)
 
+    @api.v8
+    def get_converter_for(self, field_type):
+        if self.env.__contains__('ir.qweb.field.' + field_type):
```

Comment:
Do it this way:
``` if 'ir.qweb.field.' + field_type in self.env:
            return self.env['ir.qweb.field.' + field_type]
        return self.env['ir.qweb.field'] ```
Do not call __contains__ method, you can use in operator which will do the same